src/TooltipPopoverWrapper.js

```diff
@@ -351,7 +351,7 @@ class TooltipPopoverWrapper extends React.Component {
     return (
       <PopperContent
         className={className}
-        target={this.currentTargetElement || this._targets[0]}
+        target={this._targets[0] || this.currentTargetElement}
```

Comment:
Unfortunately, this would reintroduce #1185.

Reply:
@kyletsang, I am thinking I could try to use the focused element to identify which of the _targets is associated to it. Do you think this approach might solve this issue?

Reply:
The issue here is with the button click.  `TooltipPopoverWrapper` adds the click listener to `document`, so the code below

https://github.com/reactstrap/reactstrap/blob/a97a8345f3998e9388d42550b8e70aaa62a5be50/src/TooltipPopoverWrapper.js#L166

would result in `this.currentTargetElement` being set to `e.target` due to `e.currentTarget` being null.

I don't believe we should ever be using `e.target` here, so we should modify the code to scan the current list of targets and determine which target `e.target` falls under. This should only be done if `e.currentTarget` is null.

Reply:
Hey @kyletsang, I tried implementing exactly what you said. Please, let me know if this was the implementation you were expecting.

Reply:
Looks good!  Thanks